src/group-configurations/empty-placeholder/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Add as IconAdd } from '@edx/paragon/icons';
+import { Button } from '@edx/paragon';
+
+import messages from './messages';
+
+const EmptyPlaceholder = ({ onCreateNewGroup, isExperiment }) => {
```

Comment:
[important] For the Certificates, Textbooks, and Group Configurations pages, we have the same components to display the empty state. Let's move this component to `generic` and use it for these pages.

<img width="831" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/5e0a4772-c92f-42dd-b2c0-ec402a6ce081">

Reply:
Ok for now we can resolve this conversation.
Did I understand you correctly?